### PR TITLE
fix(crux-ui): text overflow

### PR DESCRIPTION
**Pull request by bvektor**
Implemented an ExpandableText component to use where its possible for text to overflow and cause glitches.

### CODE REVIEW

File: web/crux-ui/src/components/nodes/edit-node-card.tsx [hunk 1]
```diff
@@ -159,8 +160,8 @@ const EditNodeCard = (props: EditNodeCardProps) => {
 
   return (
     <>
-      <DyoCard className={clsx(props.className, 'flex flex-row')}>
-        <div className="flex flex-col flex-grow mr-4">
+      <DyoWrap className={clsx(props.className, 'flex flex-row ')}>
```
Comment (robot9706) [inline web/crux-ui/src/components/nodes/edit-node-card.tsx:163]: Extra space after "flex-row" ?

**Review by robot9706 — CHANGES_REQUESTED** [review]

File: web/crux-ui/src/components/products/product-card.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ const ProductCard = (props: ProductCardProps) => {
             </div>
           </div>
 
-          <p className="text-md text-bright mt-4 overflow-y-auto">{product.description}</p>
+          <div className="text-md text-bright mt-4 line-clamp-2 break-words">{product.description}</div>
```
Comment (m8vago) [inline web/crux-ui/src/components/products/product-card.tsx:50]: why div?

File: web/crux-ui/src/components/products/product-details-card.tsx [hunk 3]
```diff
@@ -23,22 +20,26 @@ const ProductDetailsCard = (props: ProductDetailsCardProps) => {
...
-
   return (
     <>
       <DyoCard className={clsx(props.className ?? 'p-6', 'flex flex-col')}>
         <div className="flex flex-row">
           <Image src="/product_default.svg" alt={t('altPicture', { name: product.name })} width={100} height={100} />
 
-          <div className="flex flex-col flex-grow">
-            <DyoHeading element="h5" className="text-lg text-bright ml-4">
+          <div className="flex flex-col flex-grow w-px">
```
Comment (m8vago) [inline web/crux-ui/src/components/products/product-details-card.tsx:29]: w-px ??

**Review by polaroi8d — CHANGES_REQUESTED** [review]
Talked in-person some minor issues.

File: web/crux-ui/src/components/products/product-details-card.tsx (hunk 3 above)
Comment (m8vago) [reply web/crux-ui/src/components/products/product-details-card.tsx:29]: layout="fixed"  to the image so we don't need w-px

File: web/crux-ui/src/components/products/versions/version-details-card.tsx [hunk 4]
```diff
@@ -31,15 +25,13 @@ const VersionDetailsCard = (props: VersionDetailsCardProps) => {
...
-
-          {!overflow ? null : (
-            <DyoButton className="ml-auto my-2" text onClick={() => setShowAll(true)}>
-              {t('showAll')}
-            </DyoButton>
-          )}
+          <DyoExpandableText
+            text={version.changelog}
+            lineClamp={6}
+            className="text-md text-slate-500"
```
Comment (m8vago) [inline web/crux-ui/src/components/products/versions/version-details-card.tsx:31]: text-light from all slate-500

**Review by m8vago — CHANGES_REQUESTED** [review]

**Review by m8vago — APPROVED** [review]

**Review by robot9706 — APPROVED** [review]
🙌

**Review by polaroi8d — APPROVED** [review]
LGTM

**Pull request merged**